README.rst

```diff
@@ -120,10 +128,16 @@ Discussions take place on our mailing list.
 
 We are very interested in feedback about usability and suggestions for improvements.
 
-
 Bug Reports
 ===========
 
 Bug reports can be submitted to the issue tracker at
 
     https://github.com/statsmodels/statsmodels/issues
+
+.. |Travis Build Status| image:: https://travis-ci.org/statsmodels/statsmodels.svg?branch=master
+   :target: https://travis-ci.org/statsmodels/statsmodels
+.. |Appveyor Build Status| image:: https://ci.appveyor.com/api/projects/status/odc5c4ukhru5xicl/branch/master?svg=true
+   :target: https://ci.appveyor.com/project/josef-pkt/statsmodels/branch/master
```

Comment:
SVG image URL for project status
`https://ci.appveyor.com/api/projects/status/gx18sd2wc63mfcuc?svg=true`

but there is also a badge for master branch status:
https://ci.appveyor.com/api/projects/status/gx18sd2wc63mfcuc/branch/master?svg=true
with link in markdown
[![Build status](https://ci.appveyor.com/api/projects/status/gx18sd2wc63mfcuc/branch/master?svg=true)](https://ci.appveyor.com/project/josef-pkt/statsmodels/branch/master)